pkgs/tools/misc/mate-tweak/default.nix

```diff
@@ -0,0 +1,84 @@
+{ stdenv
+, fetchFromGitHub
+, python3
+, intltool
+, mate
+, libnotify
+, gtk3
+, gobject-introspection
+, wrapGAppsHook
+, glib
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "mate-tweak";
+  version = "20.04.0";
+
+  src = fetchFromGitHub {
+    owner = "ubuntu-mate";
+    repo = pname;
+    rev = version;
+    sha256 = "14cvh5hf61s0z75skrpz6r0z27jd01z100v2cqih2ck0pxv0483v";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    intltool
+    (python3.withPackages(ps: with ps; [ distutils_extra ]))
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    libnotify
+    glib
+    mate.mate-applets
+    mate.mate-panel
+    mate.marco
+    mate.libmatekbd
+  ];
+
+  strictDeps = false;
+
+  dontWrapGApps = true;
+
+  # Arguments to be passed to `makeWrapper`, only used by buildPython*
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  prePatch = ''
```

Comment:
Change to `postPatch`, to avoid potential conflicts if someone adds a patch later.